### PR TITLE
Migrated to event streams

### PR DESCRIPTION
**Pull request by danandrei**

### CODE REVIEW

File: server/login.js [hunk 1]
```diff
@@ -52,16 +55,20 @@ setTimeout(function() {
...
+            stream.write('You are already out. Stay out!');
+            return stream.end();
+        }
 
-    // destroy session
-    session.destroy(function () {
-        link.end();
+        // destroy session
+        session.destroy(function () {
+            stream.write();
```
Comment (IonicaBizau) [inline server/login.js:69]: Is this needed?

Comment (danandrei) [inline server/login.js:69]: stream.write ?

Comment (IonicaBizau) [inline server/login.js:69]: Yes. Isn't `stream.end()` enough? Or what does `stream.write()` do in this case?

Comment (danandrei) [inline server/login.js:69]: With only stream.end() [this](https://github.com/danandrei/login/blob/master/login.js#L167) will not get called

Comment (danandrei) [inline server/login.js:69]: You could add a stream end handler though

Comment (IonicaBizau) [inline server/login.js:69]: Got it! Nice work! :+1:

**Pull request merged**